src/app/components/auth/login/login.component.scss

```diff
@@ -0,0 +1,33 @@
+.login-container {
```

Comment:
add these to the start of the files

// Variables
$card-width: 700px;
$card-padding: 2rem;
$box-shadow: 0 4px 8px rgba(0, 0, 0, 0.8);
$mobile-breakpoint: 730px;

// Mixins
@mixin card-styles {
  padding: $card-padding;
  width: $card-width;
  box-shadow: $box-shadow;
}